pom.xml

```diff
@@ -1201,6 +1201,12 @@
                 <version>7.0.0.jre8</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.fusesource.jansi</groupId>
```

Comment:
@ChunxuTang Could you enlighten me why we need jansi here?

Reply:
Hi @yingsu00, thanks for the comment!
`presto-cli` implicitly depends on jansi. We need to explicitly marks it with a version that supports both X86 and ARM architectures.
Without this change, we'll meet some errors when running the `presto-cli`, e.g. https://github.com/prestodb/presto/issues/17150 and https://github.com/prestodb/presto/issues/17146.